package.json

```
@@ -55,13 +55,13 @@
     "configurations": {
       "ios.sim.debug": {
         "binaryPath": "ios/build/Build/Products/Debug-iphonesimulator/AllAboutOlaf.app",
-        "build": "xcodebuild -workspace ios/AllAboutOlaf.xcworkspace -scheme AllAboutOlaf -configuration Debug -destination 'platform=iOS Simulator,name=iPhone 7,OS=12.4' -derivedDataPath ios/build build",
+        "build": "xcodebuild -workspace ios/AllAboutOlaf.xcworkspace -scheme AllAboutOlaf -configuration Debug -destination 'platform=iOS Simulator,name=iPhone 7,OS=13.1' -derivedDataPath ios/build build",
```

This will need to change to iPhone 8

```suggestion
        "build": "xcodebuild -workspace ios/AllAboutOlaf.xcworkspace -scheme AllAboutOlaf -configuration Debug -destination 'platform=iOS Simulator,name=iPhone 8,OS=13.1' -derivedDataPath ios/build build",
```